tests/types/test_schema.py

```diff
@@ -226,6 +226,13 @@ def test_schema_inference_on_pandas_series():
         with pytest.raises(MlflowException, match="Unsupported numpy data type"):
             _infer_schema(pd.Series(np.array([1, 2, 3], dtype=np.float128)))
 
+    # test names
+    s = pd.Series([1, 2, 3])
+    if hasattr(s, "name"):
+        s.rename("test", inplace=True)
+        assert "test" in _infer_schema(s).input_names()
+        assert not "foo" in _infer_schema(s).input_names()
```

Comment:
Instead of asserting that `foo` is not present in the input schema, can we instead assert that the length of input names is 1?